src/tests/test_magic_cmd.py

```diff
@@ -0,0 +1,57 @@
+import sqlite3
+
+import pytest
+from IPython.core.error import UsageError
+
+
+@pytest.mark.parametrize(
+    "cell, error_type, error_message",
+    [
+        [
+            "%sqlcmd stuff",
+            UsageError,
+            "Unknown command: stuff",
```

Comment:
This error message is pretty confusing, maybe unknown argument? 
Let's keep it explicit

Reply:
fixed